cogs/general.py

```diff
@@ -2,12 +2,16 @@
 from io import StringIO
 from contextlib import suppress
 from urllib import parse
+import re
+import datetime
```

Comment:
Why is datetime imported?

Reply:
I forgot to remove it when I wanted to use datetimes to determine the message to edit, I'll remove it